examples/QIR/Optimization/Hello/Hello.csproj

```diff
@@ -1,15 +1,15 @@
-<Project Sdk="Microsoft.Quantum.Sdk/0.18.2106148911">
+<Project Sdk="Microsoft.Quantum.Sdk/0.23.198514-beta">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <QirGeneration>true</QirGeneration>
     <BuildOutputPath>$(MSBuildThisFileDirectory)build</BuildOutputPath>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Qir.Runtime" Version="0.18.2106148911-alpha" GeneratePathProperty="true" />
-    <PackageReference Include="Microsoft.Quantum.Simulators" Version="0.18.2106148911" GeneratePathProperty="true" />
+    <PackageReference Include="Microsoft.Quantum.Qir.Runtime" Version="0.22.189218-alpha" GeneratePathProperty="true" />
```

Comment:
Same here, this sticks out a bit as a mismatch, which might make it hard to clean up later.

Reply:
Thanks for the catch, fixed it.